front_end/src/app/admin/events/family-finder/household/household.component.html

```diff
@@ -0,0 +1,18 @@
+<header-event></header-event>
+
+<div class="container-fluid">
+  <preloader *ngIf="processing"></preloader>
+  <div class="mt-2 clearfix" *ngIf="eventParticipants && !processing">
```

Comment:
This stuff will be changing in the Story I am currently working so will update in that story.